### ISSUE TITLE
Issues with items becoming invalid and not allowed blocks to be broken with the items

### ISSUE THREAD

**Issue by gameplayerhd** [GC-spigot/AdvancedEnchantments#313]
Issue after enchanting items they become "invalid items", you simply can't break blocks with the items.
![image](https://user-images.githubusercontent.com/2906859/51803516-3ac48400-2256-11e9-852b-57189ad97d12.png)

Paperspigot 1.8,8
Avancedenchantment 5.1.5

Any ideas on how to resolve this?

Thanks

**Comment by GC-spigot** [comment]
Are you using ChatItem plugin to get that result?

**Comment by gameplayerhd** [comment]
Yeah, also they are not able to mine with the pickaxe after enchanting.

**Comment by gameplayerhd** [comment]
We just tested removing every single plugin beside Avancedenchantment and it would still not work. Mining only works when Avancedenchantment is disabled.

**Comment by Nightreaper77** [comment]
After some testing I've made a discovery. I re-added all the enchants in question in order to try to figure out what was causing it. I did so by using /ae reload instead of loading the plugin via restart or plugman. What I found out is that when it is loaded via /ae reload the new additions (the previously broken enchants) work fine. I tried figuring out why this was and I found out that when ae is loaded it says [AdvancedEnchantments] Successfully hooked into WorldGuard. I thought maybe there was an issue with the hook into WorldGuard so I disabled worldguard and did a plugman reload on AE and it worked. In other words, when it skips hooking into WorldGuard it works. Is there some sort of permission or flag players need to use the enchantments (obviously they have block-break already). Just to remind you, the enchants are all pickaxe based.
